src/VMware/docs/Get-AzVMwarePrivateCloudAdminCredential.md

```diff
@@ -22,16 +22,27 @@ List the admin credentials for the private cloud
 
 ## EXAMPLES
 
-### Example 1: Get admin credential of private cloud
+### Example 1: List the admin credentials for the private cloud
```

Comment:
Will there be multiple credentials for one private cloud?

Reply:
before you revert your change, can you please confirm whether there be multiple credentials for one private cloud?